### PR TITLE
Use Drush/Symfony style to improve command output and make things that are not error not look like errors

### PR DESCRIPTION
**Pull request by alexpott**
…and make things that are not error not look like errors!

New output looks like this:
Screenshot 2020-07-21 at 13 19 39

### CODE REVIEW

File: src/Commands/CreateConfigCommand.php [hunk 1]
```diff
@@ -168,7 +160,7 @@ public function createConfig($file, array $options = ['create-content' => FALSE,
       $this->io()->comment($this->contentCreator->getOutputDirectory());
```
Comment (dbosen) [inline src/Commands/CreateConfigCommand.php]: What about these comments here?

Comment (alexpott) [reply src/Commands/CreateConfigCommand.php]: Good point. Tidied them up and some other things.

File: src/Plugin/TestsiteBuilder/ConfigImporter/EntityBrowser.php [hunk 2]
```diff
@@ -1,187 +0,0 @@
...
-  protected $configReverter;
-
-  /**
-   * List of entity browser widgets supported by configuration modifications.
-   *
-   * @var array
-   */
-  protected $supportedEntityBrowserWidgets = [
-    'dropzonejs_media_entity',
-    'dropzonejs_media_entity_inline_entity_form',
```
Comment (dbosen) [inline src/Plugin/TestsiteBuilder/ConfigImporter/EntityBrowser.php]: We need this ConfigImporter at least for these dropzone widgets, to be able to upload images into the elsewhere defined media types (including their custom fields).
We lose this ability without this configuration and with the hacked in a media.type.image in src/ConfigCreator.php

File: src/ConfigCreator.php [hunk 3]
```diff
@@ -254,6 +254,12 @@ public function fixMissingConfiguration(): array {
     /** @var array $missing_configurations */
     $missing_configurations = [];
 
+    // Hack in a media.type.image.
+    // @todo create dependencies of missing configuration.
+    $missing_config_name = ConfigName::createByFullName('media.type.image');
+    $config_importer = $this->configImporterPluginManager->createInstance($missing_config_name->getType());
+    $config_importer->importConfig('', $missing_config_name->getFullName());
```
Comment (dbosen) [inline src/ConfigCreator.php]: I don't think this is sufficient for our intendet tests, see comment in src/Plugin/TestsiteBuilder/ConfigImporter/EntityBrowser.php as well.

**Review by dbosen — APPROVED** [review]

**Pull request merged**